Makefile

```diff
@@ -16,17 +16,22 @@ help:
 	@echo ""
 	@echo "  Targets"
 	@echo ""
-	@echo "    deps       Install python packages"
-	@echo "    install    Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
-	@echo "    uninstall  Uninstall scripts and $(SHAREDIR)"
-	@echo "    docker     Build Docker image"
+	@echo "    deps-ubuntu	Install system packages (on Debian/Ubuntu)"
+	@echo "    deps       	Install python packages"
+	@echo "    install    	Install the executable in $(PREFIX)/bin and the ocrd-tool.json to $(SHAREDIR)"
+	@echo "    uninstall  	Uninstall scripts and $(SHAREDIR)"
+	@echo "    docker     	Build Docker image"
 	@echo ""
 	@echo "  Variables"
 	@echo ""
 	@echo "    PREFIX  Directory to install to ('$(PREFIX)')"
 
 # END-EVAL
 
+# Install system packages (on Debian/Ubuntu)
+deps-ubuntu:
+	apt-get install -y python3 python3-venv openjdk-8-jre-headless ghostscript
```

Comment:
Would `openjdk-11-jre-headless` also work? Newer Linux distributions don't provide JRE 8.

Reply:
No, the PRImA libs still require JRE 8 AFAIK (dimly recall getting that answer directly from Christian Clausner somewhere). Unfortunately, build and deps are not well documented for PRImA's projects, but you can open an issue about it on https://github.com/PRImA-Research-Lab/prima-core-libs or https://github.com/PRImA-Research-Lab/prima-page-converter.